.env

```diff
@@ -0,0 +1,14 @@
+MENDER_SERVER_REGISTRY=docker.io
+MENDER_SERVER_ENTERPRISE_REGISTRY=registry.mender.io
+MENDER_SERVER_REPOSITORY=mendersoftware
+MENDER_SERVER_ENTERPRISE_REPOSITORY=mender-server-enterprise
+MENDER_SERVER_TAG=main
+MENDER_CLIENT_TAG=mender-master
```

Comment:
Can we have `MENDER_ARTIFACT_VERSION` here also?

Reply:
It won't have any effect as this is a variable for the `tests/run.sh` script...
Unless we source this file from the script. Yes! Let's do it :+1: